parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -350,6 +350,7 @@ ensureArtifactStreaming() {
   sudo /opt/acr/tools/overlaybd/enable-http-auth.sh
   sudo /opt/acr/tools/overlaybd/config.sh download.enable false
   sudo /opt/acr/tools/overlaybd/config.sh cacheConfig.cacheSizeGB 32
+  sudo /opt/acr/tools/overlaybd/config.sh exporterConfig.port 9863
```

Comment:
Should `exporterConfig.enable` also be set to true (https://github.com/containerd/overlaybd#:~:text=exporterConfig.enable,show%20Prometheus%20metrics.)? I am not sure how `acr/.../overlaybd/config.sh` looks, so just checking.

Reply:
Good point, yes, that's needed. Thanks! config.sh just sets the config for overlaybd. Will update it.